modes/forge/evil-collection-forge.el

```diff
@@ -44,9 +44,15 @@
   :group 'magit
   :type  'symbol)
 
+(defvar forge-add-default-bindings)
+
 ;;;###autoload
 (defun evil-collection-forge-setup ()
   "Set up `evil' bindings for `magit'."
+  (when forge-add-default-bindings
```

Comment:
I think this should be changed to `(when (boundp 'forge-add-default-bindings)`, as [the latest release tag of forge](https://github.com/magit/forge/releases/tag/v0.3.0) does not include this variable yet.

Reply:
Agreed.